src/table/sticky-columns/use-sticky-columns.ts

```diff
@@ -201,6 +201,11 @@ export function useStickyCellStyles({
     [columnId, setCell, stickyColumns.store]
   );
 
+  // Subscriptions created in the refCallback must be cancelled when the component un-mounts.
+  useEffect(() => {
+    return () => unsubscribeRef.current?.();
```

Comment:
Looks good, but I don't quite understand yet why we need to do this. Shouldn't react call the refCallback with `null` when the component unmounts, and thus already cause the subscription to be cancelled in line 162?

Reply:
Great question! That should indeed be the case.

Reply:
Updated!